cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/transaction/stream/StreamConsumerTestBase.java

```diff
@@ -254,9 +309,84 @@ public void testFIFOReconfigure() throws Exception {
   }
 
   @Test
+  public void testCombineConsumer() throws Exception {
```

Comment:
there's comments throughout the test case that describe whats going on. I think it'd be redundant to just pull those also to the top. I don't mind doing it, but I'm not sure you saw those other comments.